src/controllers/translateController.ts

```diff
@@ -56,21 +58,23 @@ const translateController = async ({
     const filesToEdit = await validateAndPromptUserJSONFiles(
       basePath,
       language.files,
-      nameOfTranslation,
+      nameOfTranslation
     );
 
     if (filesToEdit.length === 0) continue;
 
     const { text: result } = await translate(
       text,
       sourceLanguage,
-      language.name,
+      language.name
     );
 
     filesToEdit.forEach(({ file, parsedPath }) => {
       setDeepValue(file, nameOfTranslation, result);
 
-      fs.writeFileSync(parsedPath, JSON.stringify(file, null, 2) + "\n");
+      const sortedFile = sortObjectKeys(file, sort ?? "none");
```

Comment:
we can set up the default inside of the sortObjectKeys function

```suggestion
      const sortedFile = sortObjectKeys(file, sort);
```